client/extensions/woocommerce/state/action-list/test/reducer.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
I intentionally use action creators here for two reasons:

1. Writing an entire test for a one-line function that just returns an object usually comprised 1:1 of it parameters seems like a low-value test to me. It's not really benefitting much, and it's just one more thing that has to be updated next time the code is modified.

2. The sole purpose of action properties is to communicate data to reducers (and handlers). As long as the reducer understands the action properties correctly, that purpose is fulfilled. So therefore, I don't feel a need to get into the guts of this for testing purposes. Additionally, using action creators allows for a bit of resilience in the tests, as we then have the ability to make adjustments to action properties and reducers with minimal or no changes necessary to the tests.

(A really good example of this is the removal of `payload` from our actions. If we only used action creators in our tests instead of creating actions by hand, we wouldn't have had to modify our tests at all for that refactoring.)